internal/cmd/argus/grafana/public-read-access/enable/enable.go

```diff
@@ -0,0 +1,115 @@
+package enable
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/argus/client"
+	argusUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/argus/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/argus"
+)
+
+const (
+	instanceIdFlag = "instance-id"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	InstanceId string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "enable",
+		Short: "Enables public read access for Grafana on Argus instances",
+		Long: fmt.Sprintf("%s\n%s",
+			"Enables public read access for Grafana on Argus instances.",
+			"When enabled, anyone can access the Grafana dashboards without of the instance logging in. Otherwise, a login is required.",
+		),
+		Args: args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Enable public read access for Grafana on an Argus instance with ID "xxx"`,
+				"$ stackit argus grafana public-read-access enable --instance-id xxx"),
```

Comment:
would it make sense to have instance ID as an argument for all `grafana` commands? :thinking: 
The instance ID is the resource-identifier for grafana

Reply:
Adapted it